### ISSUE TITLE
Error on example

### ISSUE THREAD

**Issue by jtravs** [JuliaGraphics/QML.jl#28]
I get the following error
```
ERROR: LoadError: LoadError: InitError: No ApplicationManager instance created
 in __init__() at C:\Users\John\.julia\v0.5\QML\src\QML.jl:32
 in include_from_node1(::String) at .\loading.jl:488
 in eval(::Module, ::Any) at .\boot.jl:234
 in require(::Symbol) at .\loading.jl:415
 in include_from_node1(::String) at .\loading.jl:488
during initialization of module QML
while loading C:\Users\John\.julia\v0.5\QML\src\QML.jl, in expression starting on line 481
while loading C:\Users\John\Dropbox\qmlgui.jl, in expression starting on line 2
```

when trying to run the repl_background example. Any tips?

**Comment by barche** [comment]
If you checked out QML, be sure to also checkout CxxWrap and do
```julia
Pkg.build("QML")
```
after that. Also, this example is meant to be included from the repl using include(path-to-jl-file), you did that, right?

**Comment by jtravs** [comment]
Yes, I did that:

```
julia> Pkg.checkout("CxxWrap")
INFO: Checking out CxxWrap master...
INFO: Pulling CxxWrap latest master...
INFO: No packages to install, update or remove

julia> Pkg.checkout("QML")
INFO: Checking out QML master...
INFO: Pulling QML latest master...
INFO: No packages to install, update or remove

julia> Pkg.build("QML")
INFO: Building CxxWrap
Found Julia library at C:\Users\John\AppData\Local\Julia-0.5.0\lib\libjulia.dll.a
INFO: Building QML
Using Qt from msvc2015_64
==================================================================================[ ERROR: QML ]==================================================================================


LoadError: LoadError: InitError: No ApplicationManager instance created
during initialization of module QML
while loading C:\Users\John\.julia\v0.5\QML\src\QML.jl, in expression starting on line 481
while loading C:\Users\John\.julia\v0.5\QML\deps\build.jl, in expression starting on line 143

==================================================================================================================================================================================


=================================================================================[ BUILD ERRORS ]=================================================================================


WARNING: QML had build errors.

 - packages with build errors remain installed in C:\Users\John\.julia\v0.5
 - build the package(s) and all dependencies with `Pkg.build("QML")`
 - build a single package by running its `deps/build.jl` script
```

**Comment by barche** [comment]
Ah, sorry, on Windows it will download a binary by default, and that is not updated with the changes to the code. The repl example should work in the latest released version also (i.e. by Pkg.free on both QML and CxxWrap), or otherwise the easiest way to build binaries on Windows yourself is as follows:
- Install visual studio 2015 community edition
- Install cmake and make sure it is in the path
- Install Qt using the official installers
Next, in Julia set:
```julia
ENV["QT_ROOT"] = "C:\Qt\5.7" # or where Qt is installed
ENV["BUILD_ON_WINDOWS"] = 1
```
And then build the QML package again. Note that this needs a lot of disk space: VS takes about 10 GB and Qt about 5 GB if I remember correctly. Hopefully it will turn out to be possible to release binaries automatically with appveyor, that should eliminate this problem.

**Comment by ufechner7** [comment]
Perhaps this can be closed now that 0.8 is released?
